tools/slack/bot.py

```diff
@@ -0,0 +1,16 @@
+import os
+from slack_sdk import WebClient
+from pathlib import Path
+from dotenv import load_dotenv
+ 
+ #Create .env file and load it
+load_dotenv()
+ 
+client = WebClient(token = os.getenv('SLACK_BOT_TOKEN')) # Add SLACK_BOT_TOKEN in .env 
+ 
+try:
```

Comment:
Nice!

Next should we have the bot monitor the channel and reply 'Hello X' when X says 'Hello Hue'?

Bit similar to https://www.digitalocean.com/community/tutorials/how-to-build-a-slackbot-in-python-on-ubuntu-20-04#step-1-%E2%80%94-creating-the-slackbot-in-the-slack-ui

Then we can see how to refactor a bit to clean-up

Reply:
Sure! For the bot to monitor the slack events and respond to them, we need to look into having a web server.

Reply:
Yes, quick & dirty now and we chat in person Thursday about the long term solution

Reply:
Sounds good

Reply:
Closing this PR and opening a new one with more changes @romainr

Reply:
Also shall we change the directory name to "slackbot" ?
"slack" seems too general